### ISSUE TITLE
Why was the last major release in 2016? Homebrew is still using 1.3.1, which crashes upon launch.

### ISSUE THREAD

**Issue by jasikpark** [clementine-player/Clementine#6794]
### Before posting

Please follow the steps below and check the boxes with [x] once you did the step.

**I am sure that this may have been fixed, but if you are not supporting brew, please take clementine off of brew**
- [ ] I checked the issue tracker for similar issues
- [ ] I checked the [changelog](https://github.com/clementine-player/Clementine/blob/master/Changelog) if the issue is already resolved
- [ ] I tried the latest Clementine build from [here](https://builds.clementine-player.org/)

### System information

Please provide information about your system and the version of Clementine used.

- Operating System: macOS 10.15.6
- Clementine version: 1.3.1, installed via `brew cask install clementine`

### Expected behaviour / actual behaviour

Opening the player does not crash upon launch.

### Steps to reproduce the problem (only for bugs)

Open the player.

**Comment by matkoniecz** [comment]
Duplicate of #5958 (found by searching "abandoned")

> Why was the last major release in 2016?

No idea, but creating duplicate issues will not help.

**Comment by jasikpark** [comment]
Yeah, I was able to download the currently nightly build and it ran instead of immediately crashing, but I wish there was a way to communicate that the current reccomendation is to install the nightly since the major release from 2016 does not work on current macOS.

**Comment by matkoniecz** [comment]
Maybe Homebrew should start shipping what is a latest commit at a given time given that releases are not made anymore?
